packages/token-introspection/src/client/introspection.test.ts

```diff
@@ -98,17 +99,15 @@ describe('introspection', (): void => {
     })
   })
 
-  describe('validateTokenInfo', (): void => {
-    test('returns valid token info', async (): Promise<void> => {
+  describe('validateTokenAccess', (): void => {
```

Comment:
can you add a case where the validation fails?

Reply:
Ended up removing it, actually